src/theme.ts

```diff
@@ -1,14 +1,41 @@
 /* eslint-disable @mll-lab/no-color-literals */
 
+import {
+  red,
+  volcano,
+  orange,
+  gold,
+  yellow,
+  lime,
+  green,
+  cyan,
+  blue,
+  geekblue,
+  purple,
+  magenta,
+  grey,
+} from '@ant-design/colors';
 import { useContext } from 'react';
 import { ThemeContext } from 'styled-components';
 
-/**
- * We omit special purpose colors:
- * - 'white': Brightest possible
- * - 'black': Darkest possible
- * - 'transparent': No color
- */
+export const MLL_PALETTE = {
+  red: red.primary,
```

Comment:
This should be MLL red - `#ff000`

Reply:
I think having two slightly different shades of red on the same page looks bad regardless of their purpose.